### PR TITLE
JavaScript Milestone Project

### PR DESCRIPTION
**Pull request by AJ193**
## JavaScript Capstone
- We added a Home page that retrieves data from the Tv maze Api to show the items on the screen.
- We also used an involvement API To add likes and comments.
- We added a like button that records the Involvement API and the screen is updated.
- We added a pop-up when it is clicked it shows the info of the selected item. It also shows the comments.
- We added a form that enables users to add a comment.
- Video Link (https://drive.google.com/file/d/15ZF8Oc_Tbx7h6fFCk01r0LEswF85y1xp/view?usp=sharing)

### CODE REVIEW

File: src/modules/displayComments.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import getComments from './getComments.js';
+
+const countComments = async (data) => {
+  const length = data?.length ?? 0;
+  return length;
+};
+
```
Comment (souad988) [inline src/modules/displayComments.js:7]: well done by implementing the comments functionality , but I wonder why is this function async while it doesn't call any promises you can call it as normal function

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @AJ193 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
Good collaboration
good commit messages
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Deploying your project to make it easily accessible 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by souad988** [comment]
Also please consider this : I would suggest that you call the getdata function only once on page load and store it in a global variable or localStorage this way you reduce api calls not needed  to the movie api

File: src/modules/displayComments.js (hunk 1 above)
Comment (bronnie313) [reply src/modules/displayComments.js]: thank your @souad988 for reviewing our

**Review by omarsalem7 — APPROVED** [review]
Hi @AJ193 😊, 
# STATUS: APPROVED! 🥇 💯 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**